torch/ao/sparsity/__init__.py

```diff
@@ -5,6 +5,7 @@
 # Sparsifier
 from .sparsifier.base_sparsifier import BaseSparsifier
 from .sparsifier.weight_norm_sparsifier import WeightNormSparsifier
+from .sparsifier.nearly_diagonal_sparsifier import NearlyDiagonalSparsifier
```

Comment:
Add the following in this file:

```python
__all__ = [
    'NearlyDiagonalSparsifier',
]
```

I think that should fix the issue. I will all other imports in there in the later PRs. cc @albanD

Reply:
Actually scratch that -- We probs need to avoid exposing the `BaseSparsifier` in the `nearly_diagonal...` file